app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TweetsController < ApplicationController
+
+
+	get '/tweets' do
+		if logged_in?
+			@user = current_user
+			success
+			erb :'/tweets/tweets'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+
+	get '/tweets/new' do
+		if logged_in?
+			@user = current_user
+			erb :'/tweets/create_tweet'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+	post '/tweets/new' do
```

Comment:
Typically, following [RESTful convention](https://learn.co/tracks/full-stack-web-development-v5/sinatra/activerecord/sinatra-restful-routes), tweet creation would be posted to `/tweets` instead of `/tweets/new`.